scripts/qnx/common.mk

```diff
@@ -30,15 +30,12 @@ LDFLAGS += -Wl,--unresolved-symbols=report-all -Wl,--no-undefined -Wl,-fPIC
 
 include $(MKFILES_ROOT)/qtargets.mk
 
-CCFLAGS += -DVK_USE_PLATFORM_SCREEN_QNX=1 -Dvulkan_EXPORTS
+CCFLAGS += -DVK_USE_PLATFORM_SCREEN_QNX=1 -DVK_ENABLE_BETA_EXTENSIONS
```

Comment:
-Dvulkan_EXPORTS does nothing. So I just replaced it with VK_ENABLE_BETA_EXTENSIONS which is actually used.